### PR TITLE
vote from delegatee wallet address done

### PR DESCRIPTION
**Pull request by drsmog**
# for checking run
```npx ts-node ./scripts/operate.ts```

# note
in env file you will need to have two wallets (private key) ```PRIVATE_KEY and PRIVATE_KEY1 ```

after running you should get the following results

image

# exlanation (what we are doing in operate.ts file)

- prepare two singer (owner and other one for testing delegate)
- deploy token contract first
- mint 100 token for the owner
- delegate 100 token votes to "other" wallet address
- deploy ballot contract
- call vote (print vote power before and after)

### CODE REVIEW

**Comment by drsmog** [comment]
I've also provided simple deployments scripts for token and ballot to play around this scripts

**Comment by allen-hsu** [comment]
Hi, @drsmog  I create some function can reuse, could you and ref it to modify some? 
and put your script to `operate.ts` , because when we done part of this, and we finished the operate script part as well

File: Project/scripts/mint-and-delegate-to-address.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    process.env.PRIVATE_KEY ?? EXPOSED_KEY
+  ).connect(provider);
+
+  // signer - with private key of "Delegate To Address" wallet from env file
+  const singer2 = new ethers.Wallet(
+    process.env.PRIVATE_KEY1 || EXPOSED_KEY
+  ).connect(provider);
+
+  console.log("Signer: ", signer.address);
+  console.log("Another Signer: ", singer2.address);
```
Comment (SharanGoharKhan) [inline Project/scripts/mint-and-delegate-to-address.ts]: @drsmog `singer2` variable has a typo also it is not used anywhere else.

**Comment by drsmog** [comment]
@allen-hsu @SharanGoharKhan  @j-c17  @RussWooDev  
hey guys i've just updated PR. i've included all flow in operation.ts

so you can just test it with
```npx ts-node ./scripts/operate.ts```

in env file you will need to have two wallets (private key) ```PRIVATE_KEY and PRIVATE_KEY1 ```

after running you should get the following results

image

**Comment by drsmog** [comment]
For flow explanation I've also updated PR description

**Review by allen-hsu — APPROVED** [review]

**Comment by allen-hsu** [comment]
LGTM, thx @drsmog ~~

**Pull request merged**